### PR TITLE
Set bounding rects to actual (sub) root node.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: xray/src/bin/merge_xray_quadtrees.rs [hunk 1]
```diff
@@ -77,12 +76,18 @@ fn read_metadata_from_directories(directories: &[PathBuf]) -> Vec<Meta> {
         .collect()
 }
 
-fn get_root_node(meta: &Meta) -> Option<NodeId> {
-    meta.nodes.iter().copied().min_by_key(|node| node.level())
+fn get_root_node(meta: &Meta) -> Option<Node> {
```
Comment (krzesi-mir) [inline xray/src/bin/merge_xray_quadtrees.rs]: Let's make this function a public method of`Meta`?

Comment (feuerste) [reply xray/src/bin/merge_xray_quadtrees.rs]: Done

Comment (krzesi-mir) [reply xray/src/bin/merge_xray_quadtrees.rs]: :+1:

**Comment by feuerste** [comment]
Thanks for your review!

**Pull request merged**